internal/compiler/widgets/native/std-widgets.slint

```diff
@@ -257,3 +257,88 @@ export GridBox := GridLayout {
     spacing: NativeStyleMetrics.layout-spacing;
     padding: NativeStyleMetrics.layout-spacing;
 }
+
+export StandardTableView := Rectangle {
+    callback sort-ascending(int);
+    callback sort-descending(int);
+    
+    out property <int> current-sort-column: -1;
+    in-out property <[TableColumn]> columns;
+    in property <[[StandardListViewItem]]> rows;
+    in property <length> min-header-height: 42px;
+   
+    min-width: 400px;
+    min-height: 200px;
```

Comment:
where do these value come from?

Reply:
the values are not needed and removed now.